.github/CODEOWNERS

```diff
@@ -1 +1,22 @@
 # See for instructions on this file https://help.github.com/articles/about-codeowners/
```

Comment:
Given we have our own doc I think we should remove this link to the github codeowners docs. That link is already in our docs as well and we really want folks to read our doc when editing our codeowners files.